src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -525,8 +526,8 @@ def _evaluation_detector_conditions(
                 if condition_result.result is not None
                 and isinstance(condition_result.result, DetectorPriorityLevel)
             ]
-
-            new_priority = max(new_priority, *validated_condition_results)
+            if validated_condition_results:
+                new_priority = max(new_priority, *validated_condition_results)
```

Comment:
🤔 do we need to add this? it's just checking if there are any items in the list -- which is probably a similar amount of processing power to splat an empty list and max it.